test/software/amazon/ion/TestUtils.java

```diff
@@ -129,8 +129,7 @@ public boolean accept(File dir, String name)
 
     public static final FilenameFilter GLOBAL_SKIP_LIST =
         new FileIsNot(
-            "bad/annotationNested.10n"          // TODO amzn/ion-java#55
-            , "bad/clobWithNullCharacter.ion"               // TODO amzn/ion-java#43
+                "bad/clobWithNullCharacter.ion"               // TODO amzn/ion-java#43
```

Comment:
There appears to be an indentation inconsistency here. Please align open-paren on line 132 with the open-paren on line 133.